pages/api/s3-upload.ts

```diff
@@ -0,0 +1 @@
+export { APIRoute as default } from 'next-s3-upload';
```

Comment:
folder structure in s3 can be changed in this file if needed see 
https://next-s3-upload.codingvalue.com/s3-file-paths

Reply:
This seems like it will be useful when we add the other document types.

Reply:
+1 I'll add that as a comment in here so it's more visible